Examples/Algorithms/TrackFindingML/include/ActsExamples/TrackFindingML/SeedFilterDBScanClustering.hpp

```diff
@@ -1,61 +1,47 @@
 // This file is part of the Acts project.
 //
-// Copyright (C) 2023 CERN for the benefit of the Acts project
+// Copyright (C) 2023-2024 CERN for the benefit of the Acts project
 //
 // This Source Code Form is subject to the terms of the Mozilla Public
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 #pragma once
 
+#include "Acts/Utilities/DBScan.hpp"
+
 #include <map>
 #include <unordered_map>
 #include <vector>
 
-#include "mlpack/methods/dbscan.hpp"
-
 namespace Acts {
 
 /// Clusters seed based on their direction, their Z impact parameter and their
 /// momentum using DBScan
 ///
 /// @param input : Input parameters for the clustering (phi, eta, z, Pt)
-/// @param epsilon : Maximum distance between 2 tracks to be clustered
-/// @param minPoints : Minimum number of tracks to create a cluster
+/// @param epsilon : Maximum distance between 2 seed to be clustered
+/// @param minPoints : Minimum number of seed to create a cluster
```

Comment:
```suggestion
/// @param epsilon Maximum distance between 2 seeds to be clustered
/// @param minPoints Minimum number of seeds to create a cluster
```
could you also remove the `: ` in the line above? :)